src/olympia/amo/utils.py

```diff
@@ -585,6 +585,14 @@ def get_locale_from_lang(lang):
     if not lang or lang in ('dbg', 'dbr', 'dbl'):
         lang = 'en'
 
+    if lang == 'cak':
+        # 'cak' is not in http://cldr.unicode.org/ and therefore not supported
+        # by Babel. That breaks get_locale_from_lang() entirely, so this hack
+        # creates a fake Locale class for this locale.
+        locale = Locale('en')
```

Comment:
What's the side effect of saying that `cak` is `en` - we don't support it on addons-server?  There isn't a similar language we can use instead, I guess?

Reply:
We immediately set `locale` to the right language, so the only impact is that it will use decimal separator etc from english when formatting number - which is really the only use of that function, besides validating that the locale exists.

I couldn't find much information on how it should behave, there is [a wikipedia page](https://en.wikipedia.org/wiki/Kaqchikel_language) but it doesn't help much... Ultimately because the impact is limited I wasn't worried, so I picked a language that I knew would always work :)

Reply:
We might be able to hack this a bit better and get inspired by https://github.com/mastizada/focus-android/commit/a47e823b0f7ed7cfe641e1d511bf660376f4e7f6#diff-835b365a7ad8b4806be4fb468f33b480R58 (and see more below for the `Locale.parse` part) but I'm not sure this is overkill

Reply:
Yeah I saw that (interesting that they had the same problem we have...) but I figured it didn't add much and given our use case it was overkill